client/plasmacli/include.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/msgs"
+	"github.com/FourthState/plasma-mvp-sidechain/utils"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	ethcmn "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"math/big"
+	"strings"
+)
+
+func init() {
+	rootCmd.AddCommand(includeCmd)
+	includeCmd.Flags().Int64P(flagReplay, "r", 0, "Replay Nonce that can be incremented to allow for resubmissions of include deposit messages")
+}
+
+var includeCmd = &cobra.Command{
+	Use:   "include-deposit <nonce> <account>",
+	Short: "Include a deposit from <account> with given nonce",
+	Args:  cobra.MinimumNArgs(2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := context.NewCLIContext()
+
+		// validate addresses
+		addrToken := strings.TrimSpace(args[1])
```

Comment:
I think we should add a `--address` flag in place of args[1] here and use args[1] as account name and use `store.GetAccount(args[1])` instead